src/classes/net_sync.py

```diff
@@ -143,7 +143,8 @@ def _syncFiles(self):
             self.setProgress(self.progress)
 
         for remote_path, remote_md5 in businesslogics.items():
-            local_path = os.path.join(self.business_logic_xml_dir, os.path.basename(remote_path))
+            # local_path = os.path.join(self.business_logic_xml_dir, os.path.basename(remote_path))

```

Comment:
You can remove the comment here. We have git to remind us what the old value is. No sense cluttering up the codebase with old stuff.